auth/src/main/res/values/strings.xml

```diff
@@ -5,6 +5,8 @@
     <string name="fui_default_toolbar_title" translatable="false">@string/app_name</string>
     <string name="fui_progress_dialog_loading" translation_description="Loading text in dialog">Loading…</string>
     <string name="fui_sign_in_default" translation_description="Button text to sign in">Sign in</string>
+    <string name="fui_tos_and_pp" translation_description="Global ToS and Privacy policy message">By continuing, you are indicating that you accept our <xliff:g example="https://google.com/terms" id="tos" translation_description="">%1$s</xliff:g> and <xliff:g example="https://google.com/privacy" id="pp" translation_description="">%2$s</xliff:g>.</string>
```

Comment:
If you think this text is final let me know and I will ship them off for translation.